SConstruct

```diff
@@ -35,6 +35,19 @@ Basic usage:
     'scons sphinx' - Build the Sphinx documentation
 
     'scons doxygen' - Build the Doxygen documentation
+
+Additional help command options:
+    'scons help --options' - Print a description of user-specifiable options.
+
+    'scons help --list-options' - Print formatted list of available options.
+
+    'scons help --option=<opt>' - Print the description of a specific option
+                                  with name <opt>, for example
+                                  'scons help --option=prefix'
+
+    'scons help --defaults' - Print a description of default values. This can
+                              be combined with --option, for example
+                              'scons help --defaults --option=cc_flags'
```

Comment:
Ok - I see what you're saying. The `--defaults` flag gives values for all build platforms, whereas `--options` gives current values (which may be the default or a user-specified override), 

What about 
```
Additional help command options:
    'scons help --options' - Print a description of user-specifiable options
                             together with currently configured values.

    'scons help --list-options' - Print formatted list of available options.

    'scons help --option=<opt>' - Print the description of a specific option
                                  with name <opt> in addition to the current
                                  value; for example
                                  'scons help --option=prefix'

    'scons help --defaults' - Print a description of default values for all
                              supported platforms. This can be combined with
                              --option, for example
                              'scons help --defaults --option=cc_flags'
```

Reply:
To better document the hidden `scons help` features, I'm adding the following comment
```
# Note that 'scons help' supports additional command options that are intended for
# internal use (debugging or reST parsing of options) and thus not listed above:
#  --defaults ... list default values for all supported platforms
#  --restructured-text ... format configuration as reST
#  --dev ... add '-dev' to reST output
#  --output=<fname> ... send output to file (reST only)
```